src/components/internal/DatePicker.tsx

```diff
@@ -9,10 +9,11 @@ export interface DatePickerProps {
   value?: Date;
   onSelect: (value: Date) => void;
   disabledDays?: Modifier | Modifier[];
+  indicators?: Modifier | Modifier[];
```

Comment:
Huh, looking at the story, I expected this to be `indicators: DateOnly[]` because we're passing `indicators=[{jan1, jan2]}`...what is a `Modifier`?

Reply:
Modifier comes straight from React-Day-Picker. We could have this just be `Date[]` (I don't think Beam knows, or should care about `DateOnly`) and as I don't believe we'd need to have any special modifiers that are provided by React Date Picker outside of a date. 
Modifiers allow special syntax for doing things like a range of dates, specific days of the week, etc...

Reply:
> I don't think Beam knows, or should care about DateOnly

Yeah-ish; fwiw eventually when the new time APIs ship, `PlainDate` would be great to use here in Beam / and in Blueprint.

I guess for now I'd prefer doing `Date[]`s, b/c it keeps the API more "ours" than leaking `react-day-picker`'s types into our API.

Also can we pick a different name than `indicator`? Like...`highlightDays`? `underscoreDays`? Some semantic notion of what we're indicating?